agent/functional_tests/tests/functionaltests_test.go

```diff
@@ -670,7 +670,13 @@ func telemetryStorageStatsTest(t *testing.T, taskDefinition string) {
 		ClusterName: aws.String(newClusterName),
 	})
 	require.NoError(t, err, "Failed to create cluster")
-	defer DeleteCluster(t, newClusterName)
+	defer func() {
+		DeleteCluster(t, newClusterName)
+		cwlClient := cloudwatchlogs.New(session.New(), aws.NewConfig().WithRegion(*ECS.Config.Region))
```

Comment:
Note: if we add more container insights tests, it would be nice to see this all wrapped in a DeleteLogs() func